ui/apps/everest/src/modals/restore-db-modal/restore-db-modal.tsx

```diff
@@ -175,6 +179,10 @@ const RestoreDbModal = <T extends FieldValues>({
               {
                 label: Messages.fromPitr,
                 value: BackuptypeValues.fromPitr,
+                disabled:
+                  !!backupName &&
+                  pitrData?.latestBackupName !==
+                    watch(RestoreDbFields.backupName),
```

Comment:
Isn't this the same that comes from props?

Reply:
No it is not, because users can after change backupName to different one